drivers/ovsSwitch.go

```diff
@@ -223,6 +233,14 @@ func (sw *OvsSwitch) CreatePort(intfName string, cfgEp *OvsCfgEndpointState, pkt
 			log.Errorf("Error setting link %s up. Err: %v", ovsPortName, err)
 			return err
 		}
+
+		// Set the link mtu to 1450 to allow for 50 bytes vxlan encap
+		// (inner eth header(14) + outer IP(20) outer UDP(8) + vxlan header(8))
+		err = setLinkMtu(intfName, vxlanVethMtu)
```

Comment:
Good points.
- Yes, we need to make container MTU = host interface MTU - 50. I'm not sure how to do this yet.
- I'll make the change to set internal port MTU too.